elements/lisk-p2p/src/peer_directory/peer_book.ts

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright © 2018 Lisk Foundation
```

Comment:
we don't use 2018 license any more.. `Copyright © 2019 Lisk Foundation` please update it to 2019 and same for other files as well if it exists.